cognite/neat/utils/cdf_loaders/_data_modeling.py

```diff
@@ -87,8 +87,10 @@ def retrieve(self, ids: SequenceNotStr[str]) -> SpaceList:
     def update(self, items: Sequence[SpaceApply]) -> SpaceList:
         return self.create(items)
 
-    def delete(self, ids: SequenceNotStr[str]) -> list[str]:
-        return self.client.data_modeling.spaces.delete(ids)
+    def delete(self, ids: SequenceNotStr[str] | Sequence[Space | SpaceApply]) -> list[str]:
```

Comment:
Suggest adding a technical debt task (unless you want to tackle it immediately) on moving this to the base class.